src/Microsoft.ML.CpuMath/CpuMathUtils.netstandard.cs

```diff
@@ -300,22 +253,18 @@ public static float SumAbs(ReadOnlySpan<float> src)
         {
             Contracts.AssertNonEmpty(src);
 
-            unsafe
-            {
-                fixed (float* psrc = &MemoryMarshal.GetReference(src))
-                    return Thunk.SumAbsU(psrc, src.Length);
-            }
+            return TensorPrimitives.SumOfMagnitudes(src);
         }
 
         public static float SumAbs(float mean, ReadOnlySpan<float> src)
         {
             Contracts.AssertNonEmpty(src);
 
-            unsafe
-            {
-                fixed (float* psrc = &MemoryMarshal.GetReference(src))
-                    return (mean == 0 ? Thunk.SumAbsU(psrc, src.Length) : Thunk.SumAbsDiffU(mean, psrc, src.Length));
-            }
+            if (mean == 0)
+                return TensorPrimitives.SumOfMagnitudes(src);
```

Comment:
Most of these methods are now exactly the same in both the CpuMathUtils.netcore.cs and CpuMathUtils.netstandard.cs.  If we do my other suggestion of just using TensorPrimitives at the call sites, then these methods on CpuMathUtils can be deleted. If we don't do that suggestion, can we at least consolidate these methods that are now identical into a CpuMathUtils.cs that's used for all targets?